tools/linter/codespell/.codespell.ignorewords

```diff
@@ -1,4 +1,5 @@
 keypair
 keypairs
 als
-requestor
\ No newline at end of file
+requestor
+aParent
```

Comment:
recommend rming this, and using another name for the var that doesnt confuse `codespell` :)